### PR TITLE
edits to the original PR for supporting torchaudio HDemucs

### PR DESCRIPTION
**Pull request by adefossez**

### CODE REVIEW

**Comment by adefossez** [comment]
@skim0514 I will be holding on this one until HDemucs has landed into the next main version of torchaudio :)

File: demucs/train.py [hunk 1]
```diff
@@ -24,18 +25,40 @@
...
+class TorchHDemucsWrapper(nn.Module):
+    """Wrapper around torchaudio HDemucs implementation to provide the proper metadata
+    for model evaluation.
+    See https://pytorch.org/audio/main/tutorials/hybrid_demucs_tutorial.html"""
+
+    @capture_init
+    def __init__(self,  **kwargs):
+        super().__init__()
+        try:
+            from torchaudio.prototype.models import HDemucs as TorchHDemucs
```
Comment (carolineechen) [inline demucs/train.py:42]:
```suggestion
            from torchaudio.models import HDemucs as TorchHDemucs
```

hybrid demucs will be released as a beta feature in our upcoming release (targeting late october) 🙂 

cc @skim0514

File: demucs/train.py [hunk 2]
```diff
@@ -24,18 +25,40 @@
...
 from .solver import Solver
+from .states import capture_init
 
 logger = logging.getLogger(__name__)
 
 
+class TorchHDemucsWrapper(nn.Module):
+    """Wrapper around torchaudio HDemucs implementation to provide the proper metadata
+    for model evaluation.
+    See https://pytorch.org/audio/main/tutorials/hybrid_demucs_tutorial.html"""
```
Comment (carolineechen) [inline demucs/train.py:36]: pointing this to the stable version of the tutorial

```suggestion
    See https://pytorch.org/audio/stable/tutorials/hybrid_demucs_tutorial.html"""
```

**Review by carolineechen — COMMENTED** [review]
@adefossez @skim0514 fyi we officially released hybrid demucs in torchaudio 0.13.0 yesterday!

**Comment by adefossez** [comment]
Awesome I'll get that merged soon, thanks for the heads up !

**Pull request merged**